registration/static/js/src/registration.js

```diff
@@ -54,10 +53,45 @@ app.controller('Registration', ['$scope', '$http', 'djangoForm', function($scope
         return getFormFields().filter(key => $scope.form[key].$dirty || $scope.form[key].$message);
     };
 
+    // Returns the result of validating password using zxcvbn.
+    var validatePassword = function() {
+        var password = $scope.form.password.$viewValue;
```

Comment:
This is `null` when the user hasn't entered a password yet:

<img width="1280" alt="screen shot 2016-06-06 at 09 53 18" src="https://cloud.githubusercontent.com/assets/793379/15810808/89b71df8-2bcc-11e6-8d36-10ec54121016.png">